Userland/Applications/Browser/Tab.cpp

```diff
@@ -38,6 +39,13 @@
 
 namespace Browser {
 
+static String s_search_engine_format = "{}";
+
+static bool may_url_exist(URL& url)
+{
+    return url.is_valid() && !url.host().find(' ').has_value() && (url.host().find('.').has_value() || url.host() == "localhost");
```

Comment:
Please don't do this - if I put `http://foo` in the addressbar, I want the browser to load `http://foo`. You can instead do this *before* constructing a URL below.

Reply:
Fixed.